### DISCUSSION TITLE
CRUD - Read operation on Prisma vvcccbtbjlnigvlfdlbckuflrjjbgetigukngejviele

### DISCUSSION THREAD

**Discussion by Vishwaas** [prisma/prisma#25667] [Q&A]
### Question

This is my schema
```prisma
model User {
  internalid            String            @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
  groupid             String?           @db.Uuid
  Group               Group?          @relation("AssociatedGroup", fields: [groupid], references: [id])
}

model Group {
  id                        String    @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
  Users                  User[] @relation("AssociatedGroup")
  primaryuser_internalid String    @unique @db.Uuid
  PrimaryUser          User   @relation("PrimaryUser", fields: [primaryuser_internalid], references: [internalid])
}
```
```js 
const userAndGroup = await prisma.user.findFirst({
      where: {
        id,
      },
      include: {
        Group: true,
      },
    });
    
 ```
   Why cant I do the following?
    userAndGroup?.Group?.Users?;

I want to get all users associated to a group belonging to a particular user?

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @Vishwaas 

You're querying the `User` model and including the `Group` relation. However, you haven't explicitly included the `Users` relation of the `Group` model. In order to access `userAndGroup?.Group?.Users?`, you need to modify your query to include nested relations. 
You need to structure the query like this:

```js
const userAndGroup = await prisma.user.findFirst({
  where: {
     id
  },
  include: {
    Group: {
      include: {
        Users: true, // This will include all users associated with the group
      },
    },
  },
});
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
